docs/pages/api.rst

```diff
@@ -252,31 +252,50 @@ Other variants::
     response = mosmix.read_mosmix_s_latest()
 
 
-*******
-RADOLAN
-*******
+*****
+RADAR
+*****
 
-To use ``DWDRadolanRequest``, you have to provide a time resolution (either hourly or daily)
-and ``date_times`` (list of datetimes or strings) or a start date and end date. Datetimes
-are rounded to HH:50min as the data is packaged for this minute step. Additionally,
-you can provide a folder to store/restore RADOLAN data to/from the local filesystem.
+To use ``DWDRadarRequest``, you have to provide a ``RadarParameter``,
+which designates the type of radar data you want to obtain. There is
+radar data available at different locations within the DWD data repository:
 
-This is a short snippet which should give you an idea
-how to use ``DWDRadolanRequest`` together with ``wradlib``.
-For a more thorough example, please have a look at `example/radolan.py`_.
+- https://opendata.dwd.de/weather/radar/composit/
+- https://opendata.dwd.de/weather/radar/radolan/
+- https://opendata.dwd.de/weather/radar/radvor/
+- https://opendata.dwd.de/weather/radar/sites/
+- https://opendata.dwd.de/climate_environment/CDC/grids_germany/daily/radolan/
+- https://opendata.dwd.de/climate_environment/CDC/grids_germany/hourly/radolan/
+- https://opendata.dwd.de/climate_environment/CDC/grids_germany/5_minutes/radolan/
+
+For ``RADOLAN_CDC``-data, the time resolution parameter (either hourly or daily)
+must be specified.
+
+The ``date_times`` (list of datetimes or strings) or a ``start_date``
+and ``end_date`` parameters can optionally be specified to obtain data
+from specific points in time.
```

Comment:
`date_times` is no more. Adjust documentation.

Reply:
Are these notes for yourself @amotl ?